neutron/convert.cpp

```diff
@@ -23,18 +24,22 @@ using namespace scipp::dataset;
 
 namespace scipp::neutron {
 
-template <class T, class Op>
+template <class T, class Op, class... Args>
 T convert_generic(T &&d, const Dim from, const Dim to, Op op,
-                  const VariableConstView &arg) {
+                  const Args &... args) {
   using core::element::arg_list;
-  const auto op_ = overloaded{arg_list<double, std::tuple<float, double>>, op};
+  const auto op_ = overloaded{
+      arg_list<double,
+               std::tuple<float, std::conditional_t<true, double, Args>...>>,
```

Comment:
Not sure what the purpose of `conditional_t` is here with hardcoded `true`

Reply:
Making a `tuple<float, double, ..., double>` of the correct number of `double`. Do you know a nicer way?

Reply:
I see the context - no not in so few lines. Leave as is.